alchemiscale/storage/statestore.py

```diff
@@ -1358,7 +1364,10 @@ def get_task_transformation(
         self,
         task: ScopedKey,
         return_gufe=True,
-    ) -> Tuple[Transformation, Optional[ProtocolDAGResultRef]]:
+    ) -> Union[
+        Tuple[Transformation, Optional[ProtocolDAGResultRef]],
```

Comment:
Given the two possible return types might be good to have a quick note in docstring as to under which circumstances each is returned

Reply:
Done!